lib/phoenix/template.ex

```diff
@@ -132,6 +132,7 @@ defmodule Phoenix.Template do
       By default it raises but can be customized
       to render a particular template.
       """
+      @spec template_not_found(Template.name, map) :: no_return
```

Comment:
Done. Thanks for your patience.

On 10/03/16 16:48, José Valim wrote:
> In lib/phoenix/template.ex
> <https://github.com/phoenixframework/phoenix/pull/1583#discussion_r55699227>:
>
> > @@ -132,6 +132,7 @@ defmodule Phoenix.Template do
> >        By default it raises but can be customized
> >        to render a particular template.
> >        """
> > +      @spec template_not_found(Template.name, map) :: no_return
>
> It would be best to use |Phoenix.Template.name|. This way definitely
> works but, since it depends on the alias defined at the top, it is a
> little brittle (and we won't fail to compile if a typespec is wrong). So
> using the full name is more guaranteed. Thank you :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/phoenixframework/phoenix/pull/1583/files#r55699227>.
>